src/gleam/httpc.gleam

```diff
@@ -41,6 +44,9 @@ type BodyFormat {
 type ErlOption {
   BodyFormat(BodyFormat)
   SocketOpts(List(SocketOpt))
+  Sync(Bool)
+  // {self, once}
+  Stream(#(Atom, Atom))
```

Comment:
Use custom types rather than atoms please, to prevent invalid data from being able to be passed to the functions 🙏